packages/vite/src/node/plugins/css.ts

```diff
@@ -1316,8 +1316,12 @@ const less: StylePreprocessor = async (source, root, options, resolvers) => {
     return { code: '', errors: [normalizedError], deps: [] }
   }
 
-  const map: ExistingRawSourceMap = JSON.parse(result.map)
-  delete map.sourcesContent
+  const map: ExistingRawSourceMap = result.map
+    ? JSON.parse(result.map)
+    : undefined
+  if (map && map.sourcesContent) {
+    delete map.sourcesContent
+  }
```

Comment:
We can simplify a bit the code. LGTM otherwise. @sapphi-red would you approve this PR after checking it?
```suggestion
  const map: ExistingRawSourceMap = result.map && JSON.parse(result.map)
  if (map) {
    delete map.sourcesContent
  }
```

Reply:
nice